content/source/docs/enterprise/users-teams-organizations/teams.html.md

```diff
@@ -4,14 +4,83 @@ page_title: "Teams - Terraform Enterprise"
 sidebar_current: "docs-enterprise2-users-teams-organizations-teams"
 ---
 
+[organizations]: ./organizations.html
+[organization settings]: ./organizations.html#organization-settings
+[users]: ./users.html
+
 # Teams
 
-Teams are groups of users that reflect your company's group
-structure. Organization owners can create teams and manage user membership.
-Organization owners can give teams read, write, and admin permissions to
-workspaces.
+Teams are groups of Terraform Enterprise (TFE) [users][] within an [organization][organizations]. To delegate provisioning work, the organization's owners can grant workspace permissions to specific teams.
+
+Teams can only have permissions on workspaces within their organization, although any user in the team can belong to teams in other organizations.
+
+If a user belongs to at least one team in an organization, they are considered a member of that organization.
+
+## The Owners Team
+
+Every organization has a team named `owners`, whose members have special permissions. In TFE's documentation and UI, members of the owners team are sometimes called organization owners.
+
+An organization's creator is the first member of its owners team; other members can be added or removed in the same way as other teams. Unlike other teams, the owners team can't be deleted and can't be empty; if there is only one member, you must add another before removing them.
+
+Members of the owners team have [admin permissions](./permissions.html#admin) on every workspace in the organization. Additionally, the following tasks can only be performed by organization owners:
```

Comment:
Although it might be factually correct to say that they have admin permissions, it's not technically true, and future permissions could diverge. I think I'd say something like "have full access to every workspace".

Reply:
Ah, good point.